plugins/tradingAdvisor/baseTradingMethod.js

```diff
@@ -67,7 +68,7 @@ var Base = function(settings) {
     this.onTrade = function() {};
 
   // let's run the implemented starting point
-  this.init();
+  this.initBase();
```

Comment:
It's not working as you think, you need to await here

Reply:
Good catch @julesGoullee the strategy init function is in fact different from update and check functions since it is called from inside the constructor. I addressed it with this new commit: https://github.com/askmike/gekko/pull/2530/commits/affbb47de0db8b4f86d63abf7d21649d182fa65f

I tested an init delay of several seconds with this small sample strat, no probs so far: https://raw.githubusercontent.com/mark-sch/gekko/develop/strategies/BB-async.js

Does the new change/commit work for you?